great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -141,3 +142,269 @@ def _build_data_assistant_result(
             citation=data_assistant_result.citation,
             execution_time=data_assistant_result.execution_time,
         )
+
+    @staticmethod
+    def _build_numeric_rule() -> Rule:
+        """
+        This method builds "Rule" object focused on emitting "ExpectationConfiguration" objects for numerical columns.
+        """
```

Comment:
One small request to add a descriptive docstring to this method, otherwise LGTM

Reply:
@Shinnnyshinshin Done -- for two rules (as described in the `Scope` section)!  Thanks.